loops-arrays-objects.js

```diff
@@ -131,5 +189,41 @@ const students = [
 // ]
 // ==========================================
 
+console.log("------ OPDRACHT 3")
 
+// 	3513 = Pijlsweerd
+//  3514 = Vogelenbuurt
+//  3512 = Binnenstad
+//  3531 = Lombok
+//  3572 = Wittevrouwen
+//  3581 = Oudwijk
+//  3583 = Schildersbuurt
 
+for (let i = 0; i < students.length; i++) {
+    switch (true) {
+        case (students[i].zipCode==="3513"):
+            students[i].neighborhood = "Pijlsweerd";
+            break;
+        case (students[i].zipCode==="3514"):
+            students[i].neighborhood = "Vogelenbuurt";
+            break;
+        case (students[i].zipCode==="3512"):
+            students[i].neighborhood = "Binnenstad";
+            break;
+        case (students[i].zipCode==="3531"):
+            students[i].neighborhood = "Lombok";
+            break;
+        case (students[i].zipCode==="3572"):
+            students[i].neighborhood = "Wittevrouwen";
+            break;
+        case (students[i].zipCode==="3581"):
+            students[i].neighborhood = "Oudwijk";
+            break;
+        case (students[i].zipCode==="3583"):
+            students[i].neighborhood = "Schildersbuurt";
+            break;
+        default:
+            students[i].neighborhood = "invalid"
+    }
```

Comment:
Ik krijg altijd een beetje jeuk van `switch (true)`, want dat voelt heel erg als een if-else achtige structuur. Wat wellicht leesbaarder is, en meer waar een switch voor bedoeld is:
```suggestion
    switch (students[i].zipCode) {
        case "3513":
            students[i].neighborhood = "Pijlsweerd";
            break;
        case "3514":
            students[i].neighborhood = "Vogelenbuurt";
            break;
        case "3512":
            students[i].neighborhood = "Binnenstad";
            break;
        case "3531":
            students[i].neighborhood = "Lombok";
            break;
        case "3572":
            students[i].neighborhood = "Wittevrouwen";
            break;
        case "3581":
            students[i].neighborhood = "Oudwijk";
            break;
        case "3583":
            students[i].neighborhood = "Schildersbuurt";
            break;
        default:
            students[i].neighborhood = "invalid"
    }
```